### DISCUSSION TITLE
all sensor data in one page in grafana.

### DISCUSSION THREAD

**Discussion by wilcovh** [teslamate-org/teslamate#1177] [Ideas]
thanks for the good work and wonderfull amount of info available.
Is it possible to create a page with all the raw sensor data available. (as posted via mqtt.)
so to be able to see if e.g car is locked , windows closed without waking the car via the app?

**Comment by DrMichael** [comment]
Some status you can see in the teslamate homepage, namely locked and windows open...

MQTT delivers a lot, some of that is available in Grafana somewhere, some is irrelevant. What exactly do you have in mind?

**Reply by wilcovh** [reply]
nothing special in mind. probably gave the wrong example but I hardly use the page you mentioned. was looking for something similar to the  home assistant example. just a big readout overview of sensors/status of components. that when you want to check something you can without waking the car.

**Comment by DrMichael** [comment]
@adriankumpf Do we have those entities at all in the database? I cannot find it on first view...

**Reply by adriankumpf** [reply]
No, these ephemeral data are not stored in the database. Does it have to be Grafana though? What information is missing on the TeslaMate start/overview page?

**Reply by wilcovh** [reply]
No it does not have to be grafana, but is the easiest for me.. I was just looking for a readout of all the data on mqtt without opening the app. I'm using teslamate locally on pi and have made only grafana available through a dataplicity wormhole outside the house.
